tests/cli/test_datasets.py

```diff
@@ -145,7 +139,7 @@ def test_repository_file_to_dataset(runner, project, client):
 
     with client.with_dataset('dataset') as dataset:
         assert dataset.name == 'dataset'
-        assert '../../../a' in dataset.files
+        assert Path('../../../a') in dataset.files
```

Comment:
Maybe we should think of adding a check that the file/link actually exists in its location on the filesystem, and not only inside metadata.

Reply:
no in the scope of this PR

Reply:
And where is scope defined? 😁

Reply:
title

Reply:
😂 That's not definition. But 👌